app/controllers/todos_controller.rb

```diff
@@ -5,13 +5,9 @@ class TodosController < ApplicationController
   expose(:todos) { current_user.todos }
 
   def create
-    current_user.todos.create(todo_params)
-    redirect_to todos_path
+    todo_params = params[:todo]
+    current_user.todos.create(title: todo_params[:title])
+    redirect_to '/'
   end
 
-  private
-
-  def todo_params
-    params.require(:todo).permit(:title)
-  end
```

Comment:
I like having the whitelisting in the controller, because then you don't have whitelisting duplication between the `create` and `update` actions.